provd/proto/theme.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package theme;
+
+service ThemeService {
+    rpc SetBrightness(SetBrightnessRequest) returns (Empty);
+    rpc GetBrightness(Empty) returns (GetBrightnessResponse);
+    rpc SetAccent(SetAccentRequest) returns (Empty);
+    rpc GetAccent(Empty) returns (GetAccentResponse);
+}
+
+message Empty {}
+
+enum Brightness {
+    LIGHT = 0;
+    DARK = 1;
+}
+
+message SetBrightnessRequest {
+    Brightness brightness = 1;
+}
+
+message GetBrightnessResponse {
+    Brightness brightness = 1;
+}
+
+message SetAccentRequest {
+    string accent = 1;
```

Comment:
There's a finite number of accent colours so it might be worth putting those into an enum?